resources/views/navbar.blade.php

```diff
@@ -34,18 +41,20 @@ class="fixed top-0 z-30 w-full bg-white border-b border-theme-secondary-300 dark
                     {{ $middle }}
                 @endisset
 
-                <div class="flex justify-end">
-                    <div class="flex flex-1 justify-end items-center sm:justify-between sm:items-stretch">
+                <div class="flex justify-end items-center h-full">
+                    <div class="flex flex-1 justify-end items-center h-full">
                         @isset($desktop)
                             {{ $desktop }}
                         @else
                             @include('ark::navbar.items.desktop')
                         @endisset
                     </div>
 
-                    <div class="flex inset-y-0 right-0 items-center sm:static sm:inset-auto sm:ml-3">
+                    <span class="{{ $separatorClasses ?? 'hidden pr-2 border-l ml-7 h-7 border-theme-secondary-300 dark:border-theme-secondary-800 ' . $separatorBreakpointClass }}"></span>
+
+                    <div class="flex inset-y-0 right-0 items-center">
                         @if(is_array($navigation))
-                            @include('ark::navbar.hamburger')
+                            <x-ark-navbar-hamburger :breakpoint="$breakpoint ?? 'md'" />
```

Comment:
if you don't use `@include` you  need to check for the properties that it uses as it will no longer pass them on if not specified

Reply:
@ItsANameToo see my comment below